File: Sudoku_Java_Shell/src/examples/ConverterExample.java
package examples;

import sudoku.Converter;
import sudoku.SudokuBoardGenerator;
import sudoku.SudokuFile;
import cspSolver.ConstraintNetwork;

public class ConverterExample {

	public static void main(String[] args)
	{
		SudokuFile sf = SudokuBoardGenerator.generateBoard(16, 4, 4, 25);
		ConstraintNetwork cn = Converter.SudokuFileToConstraintNetwork(sf);
		
		System.out.println("*****************************************");
		System.out.println("below is a randomly generated SudokuFile:");
		System.out.println("*****************************************");
		System.out.println(sf);
		System.out.println("*****************************************");
		System.out.println("below is the constraint network representation of the above SudokuFile:");
		System.out.println("*****************************************");
		System.out.println(cn);


	}
}

File: Sudoku_Cpp_Shell/examples/ConverterExample.cpp
#include<iostream>
#include"../sudoku/Converter.hpp"
#include"../sudoku/SudokuBoardGenerator.hpp"
#include"../sudoku/SudokuFile.hpp"
#include"../cspSolver/ConstraintNetwork.hpp"

int main(){
	SudokuFile sf = generateBoard(9,3,3,25);
	ConstraintNetwork cn = SudokuFileToConstraintNetwork(sf);
    
    std::cout<<"*****************************************"<<std::endl;
    std::cout<<"below is a randomly generated SudokuFile:"<<std::endl;
    std::cout<<"*****************************************"<<std::endl;
    std::cout<<sf.toString()<<std::endl;
    std::cout<<"*****************************************"<<std::endl;
    std::cout<<"below is the constraint network representation of the above SudokuFile:"<<std::endl;
    std::cout<<"*****************************************"<<std::endl;
    std::cout<<cn.toString()<<std::endl;
    return 0;
}
